Ruby: Project: Caesar Cipher: Fix Ruby Caesar Cipher code example

JeffreyMPrice opened the pull request:
<!-- Thank you for taking the time to contribute to The Odin Project. In order to get this pull request (PR) merged in a reasonable amount of time, you must complete this entire template. -->

## Because
<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
There is an internal inconsistency on whether a caesar cipher should use a left shift or right shift. In the entry from Wikipedia the text and example used a left shift, while in the ruby code a right shift is used.

```ruby
  > caesar_cipher("What a string!", 5)
  => "Bmfy f xywnsl!"
```

## This PR
<!-- A bullet point list of one or more items describing the specific changes. -->

Update the project instructions to explicitly call for a right shift and note that the Wikipedia quote uses a left shift.

## Issue
<!--
If this PR closes an open issue in this repo, replace the XXXXX below with the issue number, e.g. Closes #2013.

If this PR closes an open issue in another TOP repo, replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX

If this PR does not close, but is related to another issue or PR, you can link it as above without the 'Closes' keyword, e.g. 'Related to #2013'.

_Note:_ any pull request created for an issue that already has someone else assigned **will be closed without review**.
-->
Closes #29106

## Additional Information
<!-- Any other information about this PR, such as a link to a Discord discussion. -->


## Pull Request Requirements
<!-- Replace the whitespace between the square brackets with an 'x', e.g. [x]. After you create the PR, they will become checkboxes that you can click on. -->
-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

JoshDevHub reviewed (changes requested):
@JeffreyMPrice Hello and thanks for making a PR to our curriculum.

In the future, please wait to be assigned an issue by a maintainer before beginning work on it. This allows us to discuss the problem, and it helps prevent situations where multiple people work on the same issue.

For this PR, I think the change you've made here is actually *more* confusing. The CS50 video uses a right shift, and every solution submitted by learners is also going to use a right shift.

I would accept a little note after the Wikipedia quote to specify that the project should use a right shift, but I think requiring the project to use a left shift isn't a good idea.

JeffreyMPrice commented:
Thanks @JoshDevHub ! I'll wait for the assignment next time.

I've reverted the change in output and updated the instructions to explicitly call for a right shift. I added to the quick tips that Wikipedia shows a left shift. Also threw in fixing a lower case Caesar.

You are right about all the submitted solutions! I didn't think about that.

JoshDevHub reviewed (approved):
Nice changes! Thank you for contributing.
